src/crypto/tls/handshake_server_tls13.go

```diff
@@ -377,8 +377,31 @@ func (hs *serverHandshakeStateTLS13) pickCertificate() error {
 		c.sendAlert(alertHandshakeFailure)
 		return err
 	}
+
 	hs.cert = certificate
 
+	if hs.clientHello.delegatedCredentialSupported {
+		if c.config.GetDelegatedCredential != nil {
```

Comment:
maybe reduce indentation by combining the above two conditions?

Reply:
Done ;)